js/validation.js

```diff
@@ -12,3 +12,11 @@ adForm['room_number'].addEventListener('change', (e) => {
   }
   adForm.capacity.value = POSSIBLE_NUMBER_ROOMS[0];
 });
+
+adForm['timein'].addEventListener('change', () => {
```

Comment:
Для единообразия стоит использовать точечную нотацию в таких случаях (аналогично как мы использовали **adForm.capacity**). 
Скобочная нужна, если в названии поля есть небуквенные символы (цифры, знаки, пробелы)
```suggestion
adForm.timein.addEventListener('change', () => {
```